spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubTemplate.java

```diff
@@ -65,20 +70,22 @@ public String send(final String topic, Message message) throws RuntimeException
 			try {
 				return Publisher.defaultBuilder(TopicName.create(projectId, topic))
```

Comment:
This PR is about to get merged. @viniciusccarvalho since you have the most context on what to do here, would you mind filing an issue to make sure this is addressed? Thanks!